ambry-router/src/main/java/com.github.ambry.router/PutOperation.java

```diff
@@ -334,7 +345,6 @@ void fillChunks() {
             bytesFilledSoFar += chunkToFill.fillFrom(channelReadBuffer);
             if (chunkToFill.isReady()) {
               routerCallback.onPollReady();
-              updateChunkFillerWaitTimeMetrics();
```

Comment:
why was this removed?

Reply:
its moved to fillFrom. bcoz, fillerWaitTime has to be updated when the chunk filling is complete and not when the chunk is ready. w/ encryption, chunk might move to Encrypting state before moving to ready state.